Include/descrobject.h

```diff
@@ -32,6 +32,8 @@ PyAPI_FUNC(PyObject *) PyDescr_NewGetSet(PyTypeObject *, PyGetSetDef *);
 PyAPI_FUNC(PyObject *) PyDictProxy_New(PyObject *);
 PyAPI_FUNC(PyObject *) PyWrapper_New(PyObject *, PyObject *);
 
+PyObject *_PyProperty_PropGet(PyObject *prop);
```

Comment:
Could you move this to a internal header and make it inline?
It is small and only used twice.